Volume/View Models/FollowingMagazineRow.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  FollowingMagazineRow.swift
+//  Volume
+//
+//  Created by Justin Ngai on 6/3/2022.
+//  Copyright © 2022 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct FollowingMagazineRow: View {
+    var body: some View {
+        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
```

Comment:
What does this do/is this for??

Reply:
Oops forgot to remove this. This is not used